components/Carousel/Carousel.js

```diff
@@ -1,8 +1,31 @@
 class Carousel {
+    constructor(carousel){
+        this.carousel = carousel;
+        this.leftButton = document.querySelector('.left-button');
+        this.rightButton = document.querySelector('.right-button');
+        this.images = document.querySelectorAll('img');
 
+        this.leftButton.addEventListener('click', (event) => this.left(event));
+        this.rightButton.addEventListener('click', (event) => this.right(event));
+        this.current = 0;
+        this.currentImage(); 
+
+    }
+    currentImage(){
+        this.images[0].style.display.block;
+    }
+    right(){
+        this.current++;
+        this.images[this.current].classList.add('current-image');
+    }
+    left(){ 
+        this.current--;
+        this.imagesp[this.current].classlist.add('current-image')
+    }
 }
 
-let carousel = document.querySelector();
+let carousel = document.querySelector('.carousel');
+carousel = Array.from(carousel).map(carousel => new Carousel(carousel))
```

Comment:
Amazing work on the carousel stretch here! 🔥 I like how you have the current image set with a new class name.